dvc/repo/experiments/queue/celery.py

```diff
@@ -321,6 +322,15 @@ def follow(
         for line in self.proc.follow(entry.stash_rev, encoding):
             ui.write(line, end="")
 
+        while True:
+            try:
+                exec_info = self._load_info(entry.stash_rev)
+                if exec_info.status > TaskStatus.SUCCESS:
+                    break
+            except FileNotFoundError:
+                pass
```

Comment:
The problem is that the time cost here is to wait for the data transfer to finish but the problem here is that we do not know how long it would take. 

One solution is to give a warning and exit if it hadn't finished after 5 or 10 seconds.